agent/util-scripts/test-bin/test-start-stop-tool-meister

```diff
@@ -56,17 +56,21 @@ printf -- "Exit code: %d\n" ${?} >> ${_ptms_output}
 cat -n ${_ptms_output}
 rm ${_ptms_output}
 
+function _timeout {
+    timeout --kill-after=10 --signal TERM 20 $*
```

Comment:
It's not necessary, but you're usually so careful about using `${*}` syntax, this stands out.